### ISSUE TITLE
loot de arqueologia bug en mi caso

### ISSUE THREAD

**Issue by lexintor** [RoDNet/Bugtracker-Infinium-4.0.6a#789]
Lo que me pasa que en arqueologia no puedo sacar las tablillas para colocarlas y aumentar los trocitos y aprender un objeto me sale una ventana en negro sin loot, y los demas si que les  va bien

**Comment by DatJetto** [comment]
Los objetos de las tablas van directamente a un apartado de Currency, no tienes que despojar nada 3:

**Comment by lexintor** [comment]
me refiero a las tablillas que se usan para mejorar es por ponen un ejemplo pergamino de atonato, piedra runica enana etc no me salen nuna

**Comment by ghost** [comment]
Es que eso no aparece u.U Están arreglándolo, en un post en la página del server se dice que está arqueología al 90-95% Blizzlike creo recordar, y básicamente es por eso.

Esos items o bien se cogen en subasta porque el bot que hay los mete a la venta o bien en el caso del pergamino de altonato en.. Monte Hyjal matando unos moobs.

**Comment by Dekadencee** [comment]
Efectivamente, es lo que nos falta para completar arqueología, creo haberlo avisado cuando la implementamos, de todas maneras mantenemos el reporte para informaros cuando se complete.

Un saludo
